qiskit/pulse/channels/pulse_channel_spec.py

```diff
@@ -91,9 +92,9 @@ def __init__(self,
                                         self.measures, self.acquires)
 
     @classmethod
-    def from_device(cls, backend):
+    def from_system(cls, backend):
```

Comment:
Do you think we should name this `from_backend` as this is what the method is doing?

Reply:
that's sounds reasonable.